src/app/tracking-system/tracking-system/tracking-system.component.html

```diff
@@ -278,493 +289,789 @@ <h1 class="modal-title modal-heading" id="ontologyModal" modal-heading>Taxonomy
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            @for (familyChild of family.childData; track familyChild) {
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                <li>
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    @if ((family.childData.length == 1 && familyChild.key != 'Other') || (family.childData.length > 1)) {
+                                                                                                                                                                                                                                                                                                                                                                                                    
```

Comment:
what is that?

Reply:
if you're saying about the size of html - it was such before...
should I rewrite this part in this PR?